libs/lib_instrumentation/src/main/kotlin/com/bael/dads/lib/instrumentation/di/module/MainTestActivityModule.kt

```diff
@@ -0,0 +1,26 @@
+package com.bael.dads.lib.instrumentation.di.module
+
+import com.bael.dads.lib.instrumentation.activity.MainTestActivity
+import com.bael.dads.lib.presentation.di.ActivityNameQualifier
+import com.bael.dads.lib.presentation.di.ActivityNameQualifier.Companion.ACTIVITY_MAIN
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+/**
+ * Created by stef_ang on 26/04/21.
+ */
+
+@Module
+@InstallIn(SingletonComponent::class)
```

Comment:
Sorry, could you please replace this annotation with:
@TestInstallIn(
	components = [SingletonComponent::class],
	replaces = []
)

instead? Thank you

Reply:
what is "replaces = []"?

Reply:
It replaces nothing. Since we totally use new module instead.